src/generic_core/remote-connection.ts

```diff
@@ -198,6 +198,7 @@ import uproxy_core_api = require('../interfaces/uproxy_core_api');
           },
           false
       ).then((endpoint :net.Endpoint) => {
+        log.info('SOCKS proxy listening on %1', endpoint);
```

Comment:
Would it be clearer that this is not PII if you were to just log the port?  That seems like it would still be just as useful.